doc/develop/west/zephyr-cmds.rst

```diff
@@ -281,3 +281,118 @@ Additional tips:
   separate ``west global`` command since ``global`` already searches for the
   ``GTAGS`` file starting from your current working directory. This is why you
   need to run ``global`` from inside the workspace.
+
+.. _west-patch:
+
+Working with patches: ``west patch``
+************************************
+
+The ``patch`` command allows users to apply patches to Zephyr or Zephyr modules
+in a controlled manner that makes automation and tracking easier.
+
+There are several sub-commands available:
+- ``apply``: apply a patch to the Zephyr module or a module in the workspace.
+- ``clean``: remove all patches that have been applied to the Zephyr module or a module in the workspace.
+- ``list``: list all patches that have been applied to the Zephyr module or a module in the workspace.
+- ``gh-fetch``: fetch patches from GitHub.
+
+Patches are described in a YAML file that is typically located in the ``zephyr/patches.yml`` file.
+
+A typical use case is when a project uses the :ref:`T2 Star Topology <west-t2>`
+where an external application is the manifest repository.
+
+.. code-block:: none
+  :alt: layout of a typical T2 Star Topology workspace
+
+   west-workspace/
+   ├── application/
+        ...
+   ├── west.yml
+   └── zephyr
+       ├── module.yml
+       ├── patches
+       │   └── zephyr
+       │       └── my-zephyr-change.patch
+       └── patches.yml
```

Comment:
The indentation looks off? These files should be under the `application` directory?

Reply:
All of the files are under `west-workspace`, I just didn't elaborate what is in the `application` directory (since it can vary) and left an ellipsis there instead.

Reply:
Fixed!